src/main/java/org/jenkinsci/maven/plugins/hpi/HplMojo.java

```diff
@@ -98,7 +97,12 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             mainSection.addAttributeAndCheck(new Attribute("Libraries", StringUtils.join(paths, ",")));
 
             // compute Resource-Path entry
-            mainSection.addAttributeAndCheck(new Attribute("Resource-Path",warSourceDirectory.getAbsolutePath()));
+            if (webappDirectory != null && webappDirectory.isDirectory()) {
+                mainSection.addAttributeAndCheck(new Attribute("Resource-Path",webappDirectory.getAbsolutePath()));
```

Comment:
Caused https://github.com/jenkinsci/maven-hpi-plugin/issues/299

This changed it from the source to target preventing live reload:

Before:

- WarSourceDir: /Users/timja/code/jenkins/solarized-theme-plugin/src/main/webapp

After:
- WebAppDir path: /Users/timja/code/jenkins/solarized-theme-plugin/target/solarized-theme